### PR TITLE
[ETP-560] Include `common-responsive` for Tickets v2 and RSVP v2.

### PR DESCRIPTION
**Pull request by juanfra**
🎫 [ETP-560]
🎥 https://www.loom.com/share/b2f6c8c5b0db4a86a48b479d9634e53a

Using tribe-common-responsive both in Tickets V2 and RSVP V2, so we can rely on the design system.

[ETP-560]: https://moderntribe.atlassian.net/browse/ETP-560

### CODE REVIEW

File: src/Tribe/Assets.php [hunk 1]
```diff
@@ -77,6 +77,22 @@ public function enqueue_scripts() {
...
+
+			// @todo: Remove this once we solve the common breakpoints vs container based.
+			tribe_asset(
+				$tickets_main,
+				'tribe-common-responsive',
+				'common-responsive.css',
+				[ 'tribe-common-skeleton-style' ],
+				null,
+				[
+					'groups' => [
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:89]: :no_entry_sign: Each line in an array declaration must end in a comma

<sub>hash: 6545b98b657a76bf77e1dce02516a93e</sub>

File: src/Tribe/Assets.php [hunk 2]
```diff
@@ -77,6 +77,22 @@ public function enqueue_scripts() {
...
+						'tribe-tickets-block-assets',
+						'tribe-tickets-rsvp',
+						'tribe-tickets-registration-page',
+					]
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:93]: :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 3811065f174a5aa6688ef1bda47c39c3</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 2 error(s)

File: src/Tribe/Assets.php [hunk 3]
```diff
@@ -75,6 +75,22 @@ public function enqueue_scripts() {
...
+
+			// @todo: Remove this once we solve the common breakpoints vs container based.
+			tribe_asset(
+				$tickets_main,
+				'tribe-common-responsive',
+				'common-responsive.css',
+				[ 'tribe-common-skeleton-style' ],
+				null,
+				[
+					'groups' => [
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:89]: :no_entry_sign: Each line in an array declaration must end in a comma

<sub>hash: a346e844fab2a992c36e05f56b15b468</sub>

File: src/Tribe/Assets.php [hunk 4]
```diff
@@ -75,6 +75,22 @@ public function enqueue_scripts() {
...
+						'tribe-tickets-block-assets',
+						'tribe-tickets-rsvp',
+						'tribe-tickets-registration-page',
+					]
```
Comment (tr1b0t) [inline src/Tribe/Assets.php:93]: :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 5b32de4c00ed235ddcdcf48c823355c9</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 2 error(s)

File: src/Tribe/Assets.php (hunk 4 above)
Comment (sc0ttkclark) [reply src/Tribe/Assets.php:93]:
```suggestion
					],
```

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**